testnet/tools/nns-tools/add-release-to-changelog.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+set -Eeuo pipefail
+
+NNS_TOOLS_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
+source "$NNS_TOOLS_DIR/lib/include.sh"
+
+help() {
+    print_green "
+Usage: $0 <PROPOSAL_ID>
+    PROPOSAL_ID: The ID of a recently executed Governance backend canister upgrade proposal.
+
+    Moves the pending new changelog entry from unreleased_changelog.md to CHANGELOG.md.
+
+    The commit that the proposal is baded on MUST be currently check out.
+    Otherwise, this script will do nothing, and terminate with a nonzero exit
+    status.
+
+" >&2
+    exit 1
+}
+
+PROPOSAL_ID="${1}"
+
+cd $(repo_root)
```

Comment:
Does this need quotes too? 
```suggestion
cd "$(repo_root)"
```